modules/react-native-status/desktop/rctstatus.cpp

```diff
@@ -324,6 +324,24 @@ void RCTStatus::multiAccountDeriveAddresses(QString json, double callbackId) {
         }, json, callbackId);
 }
 
```

Comment:
did you double check that all the new methods are there and the old ones are removed?

Reply:
ping @vkjr